### ISSUE TITLE
Quick download of sequence

### ISSUE THREAD

**Issue by khybiske** [SuLab/ChlamBase.org#13]
Feature request :: Possible to add little links under Gene and Gene Product to quickly copy to clipboard (or open up a FASTA window) the particular sequence? Maybe also allow the user to do that under orthologs... That's something I (and presumably others) do frequently. Would save so many steps!

**Comment by putmantime** [comment]
Currently jbrowse has a download feature that will get the DNA sequence.  When you click on the feature there is a download glyph.  I will have to think about how to get gene product sequences

**Comment by putmantime** [comment]
Added a download button to the protein sequence that will get the fasta from ncbi eutils.  Tried to add same functionality to nucleotide sequence as well but the entrez gene ids seemed to be mapped incorectly for genes in chlamydia.  Maps them to human genes, even though same api call for other bacteria work.  I have submitted this issue to NCBI's technical support.  For now the nucleotide sequence is still available from the jbrowse details view.  Will leave this issue open until the gene api call issue is sorted out
